app/models/concerns/interaction_concern.rb

```diff
@@ -0,0 +1,23 @@
+module InteractionConcern
```

Comment:
I'll admit at first I just had a regular module in `lib` because of previously working with the `gtm-utils` Ruby repo but then I googled and model concerns seemed to be the Railsy way... I followed [this](https://dev.to/software_writer/how-rails-concerns-work-and-how-to-use-them-gi6#:~:text=A%20Rails%20Concern%20is%20a,including%20class%20can%20use%20them.&text=The%20code%20inside%20the%20included,context%20of%20the%20including%20class.)

Reply:
Nice article - thanks 🎉